pom.xml

```diff
@@ -183,10 +184,17 @@
       </dependency>
 
       <!-- test dependencies -->
+<!--      <dependency>-->
+<!--        <groupId>org.sonarsource.orchestrator</groupId>-->
+<!--        <artifactId>sonar-orchestrator</artifactId>-->
+<!--        <version>${sonar.orchestrator.version}</version>-->
+<!--        <scope>test</scope>-->
+<!--      </dependency>-->
+
       <dependency>
         <groupId>org.sonarsource.orchestrator</groupId>
-        <artifactId>sonar-orchestrator</artifactId>
-        <version>${sonar.orchestrator.version}</version>
+        <artifactId>sonar-orchestrator-junit4</artifactId>
+        <version>4.0.0.404</version>
```

Comment:
So this is not linked to the sonar orchestrator version anymore?

Reply:
good catch